package/system/signify/Makefile

```diff
@@ -0,0 +1,51 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=signify
+PKG_VERSION:=29
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/aperezdc/$(PKG_NAME)/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=8d255ad88290c9b675711f47cc7336b736f6a7adcccd822a0c1a03e3d53de41c
+
+PKG_LICENSE:=ISC
+PKG_LICENSE_FILES:=
+
+PKG_MAINTAINER:=Paul Spooren <mail@aparcar.org>
+
+PKG_BUILD_PARALLEL:=1
+HOST_BUILD_PARALLEL:=1
+PKG_INSTALL:=1
+
+HOST_BUILD_DEPENDS:=libbsd/host
```

Comment:
How is this supposed to work? libbsd has no host build

Reply:
Thanks for the catch, I forgot to add it!